src/message.rs

```diff
@@ -27,6 +27,8 @@ pub fn expand(ast: &syn::DeriveInput) -> quote::Tokens {
         mod #dummy_const {
             extern crate actix;
 
+            use super::*;
```

Comment:
Maybe it should be explicit import of concrete item rather than of the whole namespace?

Reply:
Thanks, I revised as you commented.